textalyze.js

```diff
@@ -24,6 +24,15 @@ const sanitize = str => typeof str == 'string'
   ? str.toLowerCase()
   : '';
 
+const itemFrequency = array => {  
+  if (typeof array != 'object' && !array.isArray)
+    return new Map();
+
+  const map = itemCounts(array);
+  return array.reduce((frequencies, item) => frequencies
+      .set(item, (map.get(item)/array.length).toFixed(2)), new Map());
```

Comment:
We almost certainly don't want to be storing the data as strings (which is what `toFixed` will do).  We might want to convert it to a string representation when we display it, but under the hood we should store the actual frequency.